README.md

```diff
@@ -1,27 +1,92 @@
 
-## 🧞 Commands
+</div>
 
-Execute these commands from the root of your project in a terminal:
+<h1 align="center">Daytona Documentation</h1>
+<div align="center">
+Daytona's user and developer documentation.
+</div>
+</br>
+
+<p align="center">
+    <a href="https://github.com/daytonaio/docs/issues/new?assignees=&labels=bug">Report Bug</a>
+    ·
+  <a href="https://join.slack.com/t/daytonacommunity/shared_invite/zt-273yohksh-Q5YSB5V7tnQzX2RoTARr7Q">Join Our Slack</a>
+    ·
+    <a href="https://twitter.com/daytonaio">Twitter</a>
+  </p>
+
+<hr/>
+
+## Development
+The Daytona Documentation website is built using Astro and Stralight.
+The core of the website is built in React, with documentation and other content built in MDX.
+
+### Setting up a Development Environment
+The easiest way to start working on the documentation is by using a DEM such as Daytona.
+You can create a Workspace with the dependencies and configuration needed to preview the docs locally by executing the following command:
+
+```shell
+daytona create https://github.com/daytonaio/docs.git
+```
+
+Alternatively, if your local machine has Node.js and Yarn set up, you can clone this repository and execute `yarn install` to install the required dependencies.
+
+### Useful Commands
+This repository uses Yarn to handle project management.
+For convenience, a number of scripts are defined in `package.json` that can be called using Yarn:
 
 | Command                    | Action                                           |
 | :------------------------- | :----------------------------------------------- |
-| `yarn install`             | Installs dependencies                            |
 | `yarn run dev`             | Starts local dev server at `localhost:4321`      |
 | `yarn run build`           | Builds your production site to `./dist/`         |
 | `yarn run preview`         | Previews your build locally before deployment    |
 | `yarn run astro ...`       | Runs CLI commands like `astro add`, `astro check`|
 | `yarn run astro -- --help` | Provides help for using the Astro CLI            |
 
-## 👀 Looking for more information on how to structure and develop docs with Astro and Starlight?
+### Navigating the Docs
+On the filesystem, you'll find the content in `./src/content/docs` in the form of `.mdx` files.
+Daytona's documentation is organized into broad topics:
+
+*   __Configuration__   
+    Information around configuring the main components of Daytona.
+*   __Explanation__  
+    Background and conceptual information about Daytona and DEMs.
+*   __Installation__  
+    Information around installing Daytona under different environments.
+*   __Reference__  
+    Technical and architectural information about Daytona.
+*   __Usage__  
+    Practical information on using Daytona.
+
+## Contributing
+Daytona is licensed under the [Elastic License 2.0](LICENSE). If you would like to contribute to the software, you must:
```

Comment:
The License is Apache.

Reply:
Fixed in [4560ea5](https://github.com/daytonaio/docs/pull/23/commits/4560ea576d9e99e8cdce0b6dec6054ca01e6bfaf)